### PR TITLE
Add test for add and delete script.

### PR DESCRIPTION
**Pull request by saikatgpae**
# Changes made in this PR are:
 -  addNewTask test method
 -  deleteTask test method

### CODE REVIEW

File: src/common/__test__/addNew.test.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  object.description = 'Car Wash';
+  object.index = 1;
+  object.complete = false;
+  toDoContainer.innerHTML += listHTML(object);
+  expect(toDoContainer.children.length).toBe(1);
+  localStorageArray.push(object);
+  expect(localStorageArray.length).toBe(1);
+  const fetchList = toDoContainer.children[0];
+  const fetchText = fetchList.querySelector('.description');
+  expect(fetchText.value).toMatch(object.description);
```
Comment (uwadonat) [inline src/common/__test__/addNew.test.js]: - [x] Let's group your tests using the `description()` method as requested in project requirement.

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @saikatgpae,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your commit messages are very descriptive ✔️
- Descriptive pull request message ✔️
- Professional readme file ✔️
good-job-awesome


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/common/__test__/addNew.test.js (hunk 1 above)
Comment (saikatgpae) [reply src/common/__test__/addNew.test.js]: Added the same.

**Review by Ismailco — APPROVED** [review]
### Hi @saikatgpae, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**